examples/liquidity_pool/src/main.sw

```diff
@@ -0,0 +1,42 @@
+contract;
+
+use std::{
+    address::Address,
+    context::msg_amount,
+    context::call_frames::{msg_asset_id, contract_id},
+    assert::assert,
+    contract_id::ContractId,
+    token::{mint_to_address, transfer_to_output}
+};
```

Comment:
I like these ordered alphabetically. I think the formatter does that but it won't nicely break the whole `use` statement like this. Relevant: https://github.com/FuelLabs/sway/issues/1209.